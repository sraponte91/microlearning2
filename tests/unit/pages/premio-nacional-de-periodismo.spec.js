import { config, createLocalVue, mount } from '@vue/test-utils'
import VueMeta from 'vue-meta'
import PremioNacionalDePeriodismo from '@/pages/premio-nacional-de-periodismo.vue'
// create vue with vue-meta
const localVue = createLocalVue()
localVue.use(VueMeta, { keyName: 'head' })

const mock = {
    localVue,
    mocks: {}
}

const wrapper = mount(PremioNacionalDePeriodismo, mock)

describe('premio-nacional-de-periodismo.vue', () => {
    test('Is a Vue instance', () => {
        expect(wrapper.vm).toBeTruthy()
    })
    test('Renders correctly', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
    test('has correct <head> content', () => {
        expect(wrapper.vm.$metaInfo.title).toBe(config.mocks['$constants'].premioNacionalDePeriodismo.meta.title + ' - ' +
            config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'title'
        ).content).toBe(config.mocks['$constants'].premioNacionalDePeriodismo.meta.metaTitle + ' - ' +
            config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'description'
        ).content).toBe(config.mocks['$constants'].premioNacionalDePeriodismo.meta.metaDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:title'
        ).content).toBe(config.mocks['$constants'].premioNacionalDePeriodismo.meta.ogTitle + ' - ' +
            config.mocks['$constants'].home.meta.title)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:description'
        ).content).toBe(config.mocks['$constants'].premioNacionalDePeriodismo.meta.ogDescription)

        expect(wrapper.vm.$metaInfo.meta.find(
            (item) => item.hid === 'og:image'
        ).content).toBe(config.mocks['$constants'].baseURL + config.mocks['$constants'].premioNacionalDePeriodismo.meta.ogImage)
    })
})
