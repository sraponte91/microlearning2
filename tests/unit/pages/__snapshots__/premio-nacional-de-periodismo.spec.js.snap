// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`premio-nacional-de-periodismo.vue Renders correctly 1`] = `
<main>
  <div
    class="breadc-bak"
  >
    <div
      class="container margin-0-auto"
    >
      <ol
        class="breadcrumb ff-sans-r mb-2"
      >
        <li
          class="breadcrumb-item"
        >
          <a
            class=""
            href="/"
            target="_self"
          >
            Home
          </a>
        </li>
         
        <li
          class="breadcrumb-item active"
        >
          <span
            aria-current="location"
          >
            
          Premio nacional de periodismo
        
          </span>
        </li>
      </ol>
    </div>
  </div>
   
  <section
    class="article"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-8 offset-md-2"
        >
          <h1>
            Premio Nacional de Periodismo Simón Bolívar
          </h1>
           
          <img
            alt="Responsive image"
            class="img-fluid"
            src="/images/seccion-premio/premioNacionalPeriodismo_portada.jpg"
          />
           
          <div
            class="description"
          >
            <p>
              
              Esta iniciativa del Grupo Bolívar premia al periodista por su
              trabajo y beneficia a todos aquellos quienes, desde el rigor y
              la ética, ejercen el periodismo en medios radiales,
              audiovisuales, impresos y digitales.
            
            </p>
             
            <p>
              
              Por otra parte, el premio otorga becas anuales a periodistas que
              inician su carrera; además, mediante esta beca podrán acceder a
              importantes talleres de formación con maestros sobresalientes en
              el oficio.
            
            </p>
             
            <p>
              
              Para conocer más acerca de este programa, visita:
              
              <a
                href="http://premiosimonbolivar.com"
                target="_blank"
              >
                premiosimonbolivar.com
              </a>
            </p>
          </div>
           
          <div
            class="social"
          >
            <a
              href="#"
            >
              <svg
                height="32"
                overflow="visible"
                preserveAspectRatio="none"
                version="1.2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              >
                <g>
                  <path
                    d="M16.5,3.46c-0.79-0.09-1.58-0.13-2.37-0.12c-1.07-0.05-2.11,0.33-2.89,1.06c-0.76,0.81-1.15,1.89-1.08,3v2.28  H7.5v3.08h2.65v7.9h3.19v-7.9H16l0.41-3.08h-3.07v-2c-0.03-0.4,0.08-0.79,0.31-1.12c0.33-0.29,0.77-0.43,1.21-0.37h1.64V3.46  L16.5,3.46z"
                    id="facebook"
                    vector-effect="non-scaling-stroke"
                    xmlns:default="http://www.w3.org/2000/svg"
                  />
                </g>
              </svg>
            </a>
             
            <a
              href="#"
            >
              <svg
                height="32"
                overflow="visible"
                preserveAspectRatio="none"
                version="1.2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              >
                <g>
                  <path
                    d="M13.88,13.88c-1.04,1.04-2.72,1.04-3.76,0.01c0,0-0.01-0.01-0.01-0.01c-1.04-1.04-1.04-2.72-0.01-3.76  c0,0,0.01-0.01,0.01-0.01c1.04-1.04,2.72-1.04,3.76-0.01c0,0,0.01,0.01,0.01,0.01C14.83,11.13,14.74,12.78,13.88,13.88z M14.88,9.09  c-0.76-0.77-1.8-1.2-2.88-1.19c-2.26-0.01-4.09,1.82-4.1,4.08c0,0.01,0,0.01,0,0.02c-0.01,2.26,1.82,4.09,4.08,4.1  c0.01,0,0.01,0,0.02,0c2.26,0.01,4.09-1.82,4.1-4.08c0-0.01,0-0.01,0-0.02c0.01-1.09-0.42-2.14-1.2-2.9L14.88,9.09z M16.88,7.09  c-0.38-0.4-1.02-0.41-1.41-0.03s-0.41,1.02-0.03,1.41s1.02,0.41,1.41,0.03c0.2-0.19,0.31-0.46,0.31-0.73  c0.04-0.26-0.04-0.52-0.21-0.72L16.88,7.09z M12.8,5.44h1.1h1c0.36,0.01,0.72,0.04,1.07,0.1c0.25,0.04,0.5,0.1,0.74,0.19  c0.69,0.28,1.24,0.83,1.52,1.52c0.09,0.24,0.15,0.49,0.19,0.74c0.06,0.35,0.09,0.71,0.1,1.07c0,0.42,0,0.75,0,1s0,0.61,0,1.1  c0,0.48,0,0.75,0,0.8c0,0.08,0,0.31,0,0.8s0,0.85,0,1.1s0,0.58,0,1c-0.01,0.36-0.04,0.72-0.1,1.07c-0.04,0.25-0.1,0.5-0.19,0.74  c-0.28,0.69-0.83,1.24-1.52,1.52c-0.24,0.09-0.49,0.15-0.74,0.19c-0.35,0.06-0.71,0.09-1.07,0.1h-1h-3.79h-1  C8.74,18.49,8.37,18.47,8,18.43c-0.25-0.04-0.5-0.1-0.74-0.19c-0.69-0.28-1.24-0.83-1.52-1.52C5.66,16.49,5.61,16.24,5.57,16  c-0.06-0.35-0.09-0.71-0.1-1.07c0-0.42,0-0.75,0-1c0-0.25,0-0.61,0-1.1c0-0.48,0-0.75,0-0.8c0-0.08,0-0.31,0-0.8s0-0.85,0-1.1  s0-0.58,0-1C5.48,8.75,5.51,8.37,5.57,8c0.04-0.25,0.1-0.5,0.19-0.74c0.28-0.69,0.83-1.23,1.52-1.5C7.51,5.68,7.75,5.61,8,5.57  c0.35-0.06,0.71-0.09,1.07-0.1h1h2.73V5.44z M19.94,8.7c0.01-1.24-0.45-2.44-1.29-3.35c-0.91-0.84-2.11-1.3-3.35-1.29  C14.69,4.02,13.59,4,12,4S9.31,4.02,8.7,4.05c-1.24-0.01-2.44,0.46-3.35,1.3C4.51,6.26,4.05,7.46,4.06,8.7C4.02,9.31,4,10.41,4,12  s0.02,2.69,0.05,3.3c-0.01,1.24,0.45,2.44,1.29,3.35c0.91,0.84,2.12,1.3,3.36,1.29C9.31,19.98,10.41,20,12,20s2.69-0.02,3.3-0.05  c1.24,0.01,2.44-0.45,3.35-1.29c0.84-0.91,1.3-2.11,1.29-3.35C19.98,14.69,20,13.59,20,12S19.98,9.31,19.94,8.7z"
                    id="instagram"
                    vector-effect="non-scaling-stroke"
                    xmlns:default="http://www.w3.org/2000/svg"
                  />
                </g>
              </svg>
            </a>
             
            <a
              href="#"
            >
              <svg
                height="32"
                overflow="visible"
                preserveAspectRatio="none"
                version="1.2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              >
                <g>
                  <path
                    d="M4.19,9.32v10.32h3.44V9.32H4.19z M7.85,6.14C7.86,5.66,7.67,5.2,7.32,4.86c-0.37-0.34-0.87-0.52-1.38-0.5  c-0.51-0.02-1.01,0.16-1.4,0.5C4.18,5.19,3.98,5.65,3.99,6.14C3.98,6.62,4.17,7.08,4.51,7.41c0.37,0.35,0.86,0.53,1.37,0.51l0,0  C6.4,7.94,6.91,7.76,7.29,7.41c0.35-0.33,0.55-0.79,0.53-1.27l0,0H7.85z M19.99,13.73c0.09-1.25-0.3-2.48-1.08-3.46  c-0.74-0.79-1.78-1.22-2.86-1.18c-0.4,0-0.8,0.05-1.19,0.16c-0.32,0.1-0.61,0.25-0.87,0.45c-0.21,0.16-0.41,0.34-0.59,0.54  c-0.16,0.18-0.3,0.38-0.43,0.58l0,0v-1.5H9.52v0.5c0,0.33,0,1.36,0,3.08s0,3.97,0,6.74h3.47v-5.76c-0.03-0.3,0-0.59,0.07-0.88  c0.14-0.35,0.37-0.65,0.66-0.89c0.31-0.25,0.7-0.37,1.1-0.36c0.52-0.04,1.02,0.2,1.32,0.62c0.31,0.52,0.45,1.11,0.42,1.71v5.52h3.43  V13.73L19.99,13.73z"
                    id="linkedin"
                    vector-effect="non-scaling-stroke"
                    xmlns:default="http://www.w3.org/2000/svg"
                  />
                </g>
              </svg>
            </a>
             
            <a
              href="#"
            >
              <svg
                height="32"
                overflow="visible"
                preserveAspectRatio="none"
                version="1.2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink"
              >
                <g>
                  <path
                    d="M20.2,6.92c-0.62,0.27-1.27,0.45-1.94,0.52c0.71-0.41,1.24-1.07,1.48-1.85c-0.66,0.39-1.38,0.67-2.13,0.81  c-1.27-1.36-3.41-1.43-4.76-0.15c-0.03,0.02-0.05,0.05-0.08,0.07c-0.65,0.62-1.01,1.48-1,2.38c0,0.26,0.03,0.52,0.08,0.77  C10.51,9.41,9.19,9.05,8,8.43C6.82,7.84,5.78,7.01,4.94,6C4.35,7.02,4.33,8.27,4.89,9.3C5.16,9.77,5.54,10.16,6,10.45  C5.46,10.43,4.93,10.27,4.46,10l0,0c-0.01,0.78,0.26,1.54,0.77,2.14c0.49,0.6,1.17,1.01,1.93,1.16c-0.29,0.07-0.58,0.11-0.88,0.11  c-0.21,0-0.43-0.02-0.64-0.05c0.44,1.37,1.7,2.3,3.14,2.33c-1.17,0.97-2.65,1.5-4.17,1.51c-0.27,0.02-0.54,0.02-0.81,0  c1.55,0.99,3.36,1.5,5.2,1.46c1.14,0.01,2.28-0.19,3.36-0.57c0.96-0.34,1.86-0.85,2.64-1.51c0.74-0.63,1.39-1.37,1.92-2.19  c0.52-0.8,0.93-1.67,1.2-2.59c0.26-0.88,0.4-1.79,0.4-2.7c0-0.19,0-0.34,0-0.44C19.18,8.18,19.75,7.59,20.2,6.92L20.2,6.92z"
                    id="twitter"
                    vector-effect="non-scaling-stroke"
                    xmlns:default="http://www.w3.org/2000/svg"
                  />
                </g>
              </svg>
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
   
  <section
    class="article-slider"
  >
    <div
      class="container"
    >
      <div>
        <h2>
          Otros artículos interesantes
        </h2>
         
        <div>
          <div
            class="VueCarousel"
          >
            <div
              class="VueCarousel-wrapper"
            >
              <div
                class="VueCarousel-inner"
                style="transform: translate(0px, 0); transition: 0.5s ease transform; flex-basis: 0px; visibility: hidden; height: auto;"
              >
                <div
                  aria-hidden="true"
                  class="VueCarousel-slide"
                  role="tabpanel"
                  tabindex="-1"
                >
                  <div
                    class="position-relative box bg-white"
                  >
                    <img
                      alt=""
                      class="img-fluid"
                      src="/images/otros-interesantes.jpeg"
                    />
                     
                    <div
                      class="tags"
                    >
                      Categoría #Cultivarte #Apoyo
                    </div>
                     
                    <h3
                      class="title"
                    >
                      Título de la noticia
                    </h3>
                     
                    <div
                      class="date"
                    >
                      Septiembre 16 - 2021
                    </div>
                     
                    <div
                      class="desc"
                    >
                      
                  Synergistically redefine high-quality ideas after transparent strategic theme.
                
                    </div>
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  class="VueCarousel-slide"
                  role="tabpanel"
                  tabindex="-1"
                >
                  <div
                    class="position-relative box bg-white"
                  >
                    <img
                      alt=""
                      class="img-fluid"
                      src="/images/otros-interesantes.jpeg"
                    />
                     
                    <div
                      class="tags"
                    >
                      Categoría #Cultivarte #Apoyo
                    </div>
                     
                    <h3
                      class="title"
                    >
                      Título de la noticia
                    </h3>
                     
                    <div
                      class="date"
                    >
                      Septiembre 16 - 2021
                    </div>
                     
                    <div
                      class="desc"
                    >
                      
                  Synergistically redefine high-quality ideas after transparent strategic theme.
                
                    </div>
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  class="VueCarousel-slide"
                  role="tabpanel"
                  tabindex="-1"
                >
                  <div
                    class="position-relative box bg-white"
                  >
                    <img
                      alt=""
                      class="img-fluid"
                      src="/images/otros-interesantes.jpeg"
                    />
                     
                    <div
                      class="tags"
                    >
                      Categoría #Cultivarte #Apoyo
                    </div>
                     
                    <h3
                      class="title"
                    >
                      Título de la noticia
                    </h3>
                     
                    <div
                      class="date"
                    >
                      Septiembre 16 - 2021
                    </div>
                     
                    <div
                      class="desc"
                    >
                      
                  Synergistically redefine high-quality ideas after transparent strategic theme.
                
                    </div>
                  </div>
                </div>
                <div
                  aria-hidden="true"
                  class="VueCarousel-slide"
                  role="tabpanel"
                  tabindex="-1"
                >
                  <div
                    class="position-relative box bg-white"
                  >
                    <img
                      alt=""
                      class="img-fluid"
                      src="/images/otros-interesantes.jpeg"
                    />
                     
                    <div
                      class="tags"
                    >
                      Categoría #Cultivarte #Apoyo
                    </div>
                     
                    <h3
                      class="title"
                    >
                      Título de la noticia
                    </h3>
                     
                    <div
                      class="date"
                    >
                      Septiembre 16 - 2021
                    </div>
                     
                    <div
                      class="desc"
                    >
                      
                  Synergistically redefine high-quality ideas after transparent strategic theme.
                
                    </div>
                  </div>
                </div>
                <div
                  class="VueCarousel-slide VueCarousel-slide-active"
                  role="tabpanel"
                  tabindex="-1"
                >
                  <div
                    class="position-relative box bg-white"
                  >
                    <img
                      alt=""
                      class="img-fluid"
                      src="/images/otros-interesantes.jpeg"
                    />
                     
                    <div
                      class="tags"
                    >
                      Categoría #Cultivarte #Apoyo
                    </div>
                     
                    <h3
                      class="title"
                    >
                      Título de la noticia
                    </h3>
                     
                    <div
                      class="date"
                    >
                      Septiembre 16 - 2021
                    </div>
                     
                    <div
                      class="desc"
                    >
                      
                  Synergistically redefine high-quality ideas after transparent strategic theme.
                
                    </div>
                  </div>
                </div>
                <div
                  class="VueCarousel-slide VueCarousel-slide-active"
                  role="tabpanel"
                  tabindex="-1"
                >
                  <div
                    class="position-relative box bg-white"
                  >
                    <img
                      alt=""
                      class="img-fluid"
                      src="/images/otros-interesantes.jpeg"
                    />
                     
                    <div
                      class="tags"
                    >
                      Categoría #Cultivarte #Apoyo
                    </div>
                     
                    <h3
                      class="title"
                    >
                      Título de la noticia
                    </h3>
                     
                    <div
                      class="date"
                    >
                      Septiembre 16 - 2021
                    </div>
                     
                    <div
                      class="desc"
                    >
                      
                  Synergistically redefine high-quality ideas after transparent strategic theme.
                
                    </div>
                  </div>
                </div>
              </div>
            </div>
             
            <div
              class="VueCarousel-navigation"
              data-v-453ad8cd=""
            >
              <button
                aria-label="Previous page"
                class="VueCarousel-navigation-button VueCarousel-navigation-prev"
                data-v-453ad8cd=""
                style="padding: 8px; margin-right: -8px;"
                tabindex="0"
                type="button"
              >
                ◀
              </button>
               
              <button
                aria-label="Next page"
                class="VueCarousel-navigation-button VueCarousel-navigation-next VueCarousel-navigation--disabled"
                data-v-453ad8cd=""
                style="padding: 8px; margin-left: -8px;"
                tabindex="-1"
                type="button"
              >
                ▶
              </button>
            </div>
             
            <!---->
          </div>
        </div>
      </div>
    </div>
  </section>
</main>
`;
